### PR TITLE
feat(x/foundation): remove unnecessary gov-mint feature

### PR DESCRIPTION
**Pull request by jaeseung-bae**
## Description

closes: #843

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [x] I have updated the documentation accordingly.
- [x] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

File: x/foundation/keeper/keys.go [hunk 1]
```diff
@@ -22,8 +22,6 @@ var (
 	grantKeyPrefix = []byte{0x20}
 
 	poolKey = []byte{0x30}
-
-	govMintKey = []byte{0x40}
```
[the remark this answers is not in the document]
Comment (jaeseung-bae) [reply x/foundation/keeper/keys.go:26]: I fixed it. I commented and made it deprecated.

[the remark this answers is not in the document]
Comment (0Tech) [reply x/foundation/keeper/keys.go:26]: You can comment out the key, so the linter would be happy.

[the remark this answers is not in the document]
Comment (0Tech) [reply x/foundation/keeper/keys.go:26]: > You can comment out the key, so the linter would be happy.

Never mind. It should be fine as is.

**Review by 0Tech — APPROVED** [review]
LGTM

**Review by tkxkd0159 — APPROVED** [review]
LGTM

**Pull request merged**